test/e2e/framework/node/resource.go

```diff
@@ -38,6 +38,9 @@ import (
 	clientretry "k8s.io/client-go/util/retry"
 	e2elog "k8s.io/kubernetes/test/e2e/framework/log"
 	"k8s.io/kubernetes/test/e2e/system"
```

Comment:
It is better to move the TODO before this line because it also needs to be removed.

Reply:
good catch.Thanks

Reply:
done

Reply:
/cc @oomichi
hi,how about this now?